lambdas/ecs-events-to-slack/notify.py

```diff
@@ -0,0 +1,142 @@
+import json
+import boto3
+import requests
+from dateutil.parser import parse
+from botocore.exceptions import ClientError
+import os
+
+
+def message_formatter(event, region):
+    message_color = "warning"
+
+    event_id = event["id"]
+    event_time = event["time"]
+    parsed_time = parse(event_time)
+    formatted_time = str(parsed_time.replace(microsecond=0))
+
+    event_detail = event["detail"]
+    container_name = event_detail["overrides"]["containerOverrides"][0]["name"]
+    task_arn = event_detail["taskArn"]
+    task_id = task_arn.split("/")[1]
+    version = event_detail["version"]
+    attachment_status = event_detail["attachments"][0]["status"]
+
+    env_vars = event_detail["overrides"]["containerOverrides"][0]["environment"]
+    env_vars_list = []
+    for env_var in env_vars:
+        env_var_formatted = "%s=\"%s\""
+        % (env_var["name"], env_var["value"])
+        env_vars_list.append(env_var_formatted)
+    env_vars_formatted = "\n".join(env_vars_list)
+
+    logs_url = "https://%s.console.aws.amazon.com/cloudwatch/home?region=%s#logEventViewer:group=%s;stream=ecs/%s/%s"
```

Comment:
Okay sorry these are all nits but I slightly prefer this style of string formatting: https://docs.python.org/3/library/stdtypes.html#str.format
So you could do something like "https://{region}.console.aws.amazon.com/cloudwatch/home?region={region}#logEventViewer:group={container_name};stream=ecs/{container_name}/{task_id}".format(region=region, container_name=container_name, task_id=task_id)

actually I don't remember the actual syntax but I think it's something like that :)